tests/test_base.py

```diff
@@ -35,6 +35,20 @@ def test_passwd_present(auto_container):
     assert auto_container.connection.file("/etc/passwd").exists
 
 
+@pytest.mark.skipif(
+    OS_VERSION not in ("tumbleweed",),
+    reason="requires glibc-locale-base installed",
```

Comment:
Wouldn't it make sense to install `glibc-locale-base` on non Tumbleweed?

Reply:
yes, it would totally make sense to ship base containers that are based on SLES components rather than kubic-locale-archive. but that's a change on dockerfile-generator..